public/assets/js/scripts.js

```diff
@@ -146,4 +146,47 @@ $(document).ready(function () {
     });
   });
 
+  // use ajax to update select options on index page
+  $('#state').on('change', function () {
+    var stateID = $(this).val();
+    if (stateID) {
+      $.ajax({
+        type: 'POST',
+        url: './includes/find-house.php',
+        data: {
+          state_id: stateID
+        },
+        success: function(html){
+          $('#area').html(html);
+          $('#type').html('<option value="">Select area first</option>');
```

Comment:
I think it will be a better user experience the `type select` should also be disabled when the `area select` is active and yet to have a selected option.